architecture/2024-08-controller-component-threat-model.md

```diff
@@ -0,0 +1,186 @@
+# Title
+
+- **Author**: ytimocin
+
+## Overview
+
+This document aims to provide a threat model for the Radius Controller component. In this threat model, you will find potential security threats to the Controller component, an important part of Radius, and possible mitigations to the risks they pose. The document includes an analysis of the system, its assets, security objectives, identified threats, and recommended security measures to protect the system.
+
+This document will not include designs for the proposed measures to each threat.
+
```

Comment:
Do we need a ## Terms and definitions section? eg. for ucpd

Reply:
Yes, we do. I will add that one too. Thanks for the reminder.

Reply:
please add definition of UCPD and any other acronyms used in the doc.